utils/utils.go

```
@@ -0,0 +1,13 @@
+package utils
+
+import (
+	"os/exec"
+	"strings"
+)
+
+func GetPrjPath() string {
+	path_cmd := exec.Command("/bin/sh", "-c", "echo $GOPATH")
+	path, _ := path_cmd.Output()
```

Any reason to ignore the error?